sql/updates/world/2016_01_01_gothik.sql

```diff
@@ -0,0 +1,93 @@
+INSERT INTO `creature_text` (`entry`,`groupid`,`id`,`text`,`type`,`probability`,`sound`,`BroadcastTextId`,`TextRange`,`comment`) VALUES
+(16060,0,0,"Foolishly you have sought your own demise.",14,100,8807,13030,3,"Gothik SAY_INTRO_1"),
+(16060,1,0,"Brazenly you have disregarded powers beyond your understanding.",14,100,0,13031,3,"Gothik SAY_INTRO_2"),
+(16060,2,0,"You have fought hard to invade the realm of the harvester.",14,100,0,13032,3,"Gothik SAY_INTRO_3"),
+(16060,3,0,"Now there is only one way out- to walk the lonely path of the damned.",14,100,0,13033,3,"Gothik SAY_INTRO_4"),
+(16060,4,0,"I have waited long enough. Now you face the harvester of souls.",14,100,8808,13028,3,"Gothik SAY_PHASE_TWO"),
+(16060,5,0,"I... am... undone.",14,100,8805,13026,3,"Gothik SAY_DEATH"),
+(16060,6,0,"Death is the only escape!",14,20,8806,13027,3,"Gothik SAY_KILL"),
+(16060,7,0,"%s teleports into the fray!",41,100,0,32306,3,"Gothik EMOTE_PHASE_TWO"),
+(16060,8,0,"The central gate opens!",41,100,0,32307,3,"Gothik EMOTE_GATE_OPENED");
+
+-- minion AI
+UPDATE `creature_template` SET `ScriptName`="npc_gothik_minion_LivingTrainee" WHERE `entry`=16124;
+UPDATE `creature_template` SET `ScriptName`="npc_gothik_minion_LivingKnight" WHERE `entry`=16125;
+UPDATE `creature_template` SET `ScriptName`="npc_gothik_minion_LivingRider" WHERE `entry`=16126;
+UPDATE `creature_template` SET `ScriptName`="npc_gothik_minion_SpectralTrainee" WHERE `entry`=16127;
+UPDATE `creature_template` SET `ScriptName`="npc_gothik_minion_SpectralKnight" WHERE `entry`=16148;
+UPDATE `creature_template` SET `ScriptName`="npc_gothik_minion_SpectralRider" WHERE `entry`=16150;
+UPDATE `creature_template` SET `ScriptName`="npc_gothik_minion_SpectralHorse" WHERE `entry`=16149;
+
+-- minion difficulty spell entries
+DELETE FROM `spelldifficulty_dbc` WHERE `id` IN (55604,27825,27831,27989,56408,27993,55606,27994,55648);
+INSERT INTO `spelldifficulty_dbc` (`id`,`spellid0`,`spellid1`) VALUES
+(55604,55604,55645), -- death plague
+(27831,27831,55638), -- shadow bolt volley
+(27989,27989,56407), -- arcane explosion
+(55606,55606,55608), -- unholy aura
+(27994,27994,55646), -- drain life
+(55648,55648,27995); -- unholy frenzy
+
+-- rider damage aura + visual aura
+DELETE FROM `creature_template_addon` WHERE `entry` IN (16126,16148,16150);
+INSERT INTO `creature_template_addon` (`entry`,`mount`,`auras`) VALUES
+(16126,25278,"55606"),
+(16148,    0,"10848"),
+(16150,    0,"55606 10848");
+
+-- trigger AI
+UPDATE `creature_template` SET `ScriptName`="npc_gothik_trigger" WHERE `entry`=16137;
+-- re-do spawn locations for triggers
+DELETE FROM `creature` WHERE `id`=16137;
+INSERT INTO `creature` (`guid`,`id`,`map`,`spawnMask`,`phaseMask`,`position_x`,`position_y`,`position_z`,`orientation`,`spawntimesecs`,`spawndist`,`movementtype`) VALUES
+                       (@CGUID+0,16137,533,3,1,2643.72998 ,-3399.679932,285       ,0,0,0,0), -- living   side soul  trigger (south)
```

Comment:
it seems like an attempt to make the queries look nice by lining up the parentheses.
But yeah. Not TC codestyle (or any regular codestyle, for that matter).